README.md

```diff
@@ -1,6 +1,6 @@
 # node-oracledb version 1.4
 
-## <a name="about"></a> 1. About node-oracledb
+## <a name="about"></a> About node-oracledb
```

Comment:
It's not a huge issue but in general, when I'm developing for a particular platform, I try to follow the conventions of that platform. This makes it easier for the community to understand. In this case, on the npm platform, if you look at the top 15 libraries on the [npm homepage](https://www.npmjs.com/) you can see that none of these 15 libraries number their bullet points. I think it's a markdown convention really. But like I said it's, only a small thing.